app/Transformers/AlbumsTransformer.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Transformers;
+
+use App\Album;
+use League\Fractal\TransformerAbstract;
+
+class AlbumsTransformer extends TransformerAbstract
```

Comment:
Each of these `Transformer` classes should be singular. It's only job is to transform a single entity. 

`AlbumTransformer` for example.